client/data/emails/use-resend-verify-email-forward-mutation.ts

```diff
@@ -0,0 +1,83 @@
+import { useTranslate } from 'i18n-calypso';
+import { createElement } from 'react';
+import { useMutation } from 'react-query';
+import { useDispatch } from 'react-redux';
+import { getEmailForwardAddress } from 'calypso/lib/emails';
+import { CALYPSO_CONTACT } from 'calypso/lib/url/support';
+import wp from 'calypso/lib/wp';
+import { errorNotice, successNotice } from 'calypso/state/notices/actions';
+import type { Mailbox } from './types';
+import type { UseMutationOptions } from 'react-query';
+
+const MUTATION_KEY = 'reverifyEmailForward';
+
+/**
+ * Manually trigger a new verification email to an email forward receiver
+ *
+ * @param domainName The domain name of the mailbox
+ * @param mutationOptions Mutation options passed on to `useMutation`
+ * @returns Returns the result of the `useMutation` call
+ */
+export default function useResendVerifyEmailForwardMutation(
+	domainName: string,
+	mutationOptions: Omit< UseMutationOptions< any, unknown, Mailbox >, 'mutationFn' > = {}
+) {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	const suppliedOnError = mutationOptions.onError;
+	const suppliedOnSuccess = mutationOptions.onSuccess;
+
+	mutationOptions.mutationKey = MUTATION_KEY;
+
+	mutationOptions.onSuccess = ( data, emailForward, context ) => {
+		suppliedOnSuccess?.( data, emailForward, context );
+
+		const destination = getEmailForwardAddress( emailForward );
+
+		const successMessage = translate(
+			'Successfully sent confirmation email for %(email)s to %(destination)s.',
+			{
+				args: {
+					email: emailForward.mailbox + '@' + domainName,
+					destination,
+				},
+			}
+		);
+
+		dispatch(
+			successNotice( successMessage, {
+				duration: 5000,
```

Comment:
I think it would be great to be consistent - here the duration was set to 5 seconds but in other places we set it to 7 seconds.

Reply:
You're totally right. This was copied from the Redux reducer, but there's no reason not to be consistent about it. 

Makes me feel like there ought to be standardized delay across all of Calypso 🤔